src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -10,6 +10,20 @@ import s from "./ThreeParagraphBlock.module.css";
 const ParagraphPropType = PropTypes.shape({
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
+  button: PropTypes.oneOfType([
+    PropTypes.exact({
+      text: PropTypes.string,
+      externalLink: PropTypes.string,
+    }),
+    PropTypes.exact({
+      text: PropTypes.string,
+      internalLink: PropTypes.string,
+    }),
+    PropTypes.exact({
+      text: PropTypes.string,
+      onClick: PropTypes.func,
+    }),
+  ]),
```

Comment:
There will be a common PropType file right? @richardxia @cw323  so will need to change this in the future then.

Reply:
Yep! I'll be working on that after the `OneParagraphBlock` component.